packages/mermaid/src/mermaid.spec.ts

```diff
@@ -48,11 +48,32 @@ describe('when using mermaid and ', function () {
       const node = document.createElement('div');
       node.appendChild(document.createTextNode('graph TD;\na;'));
 
-      mermaid.initThrowsErrors(undefined, node);
+      await mermaid.initThrowsErrors(undefined, node);
```

Comment:
By the way, we should put into the `CHANGELOG.md`/Release notes that this function is now `async`.

Reply:
Can you add that to this PR?

Reply:
I'm not really sure where to put it.

I could put it into `CHANGELOG.md`, but that's super out-dated, see https://github.com/mermaid-js/mermaid/issues/3022

Maybe we should use something like [`gren changelog`](https://github.com/github-tools/github-release-notes) to automatically generate the CHANGELOG.md from the GitHub release notes in https://github.com/mermaid-js/mermaid/releases, but then there'll be no point in manually editing the CHANGELOG.md, since it would just be overwritten by whatever is written in `https://github.com/mermaid-js/mermaid/releases`